src/ui/public/render_directive/render_directive.js

```diff
@@ -6,7 +6,8 @@ module.directive('renderDirective', function () {
   return {
     restrict: 'E',
     scope: {
-      'definition': '='
+      'definition': '=',
+      'scope': '=?'
```

Comment:
I tried it out.  It works but feels a little awkward.  I have to define the attributes in both the definition and in the render-directive html.